core/test/audits/dobetterweb/uses-http2-test.js

```diff
@@ -5,26 +5,62 @@
  */
 
 import UsesHTTP2Audit from '../../../audits/dobetterweb/uses-http2.js';
-import {NetworkRecords} from '../../../computed/network-records.js';
 import {networkRecordsToDevtoolsLog} from '../../network-records-to-devtools-log.js';
-import {getURLArtifactFromDevtoolsLog, readJson} from '../../test-utils.js';
+import {createTestTrace} from '../../create-test-trace.js';
 
-const trace = readJson('../../fixtures/traces/progressive-app-m60.json', import.meta);
-const devtoolsLog = readJson('../../fixtures/traces/progressive-app-m60.devtools.log.json', import.meta);
+const baseResources = [{
+  url: 'https://www.example.com/',
+  priority: 'High',
+  rendererStartTime: 0,
+  networkEndTime: 100,
+  protocol: 'HTTP/1.1',
+},
+{
+  url: 'https://www.example.com/2',
+  priority: 'High',
+  rendererStartTime: 200,
+  networkEndTime: 300,
+  protocol: 'HTTP/1.1',
+},
+{
+  url: 'https://www.example.com/3',
+  priority: 'High',
+  rendererStartTime: 400,
+  networkEndTime: 500,
+  protocol: 'HTTP/1.1',
+}];
+
+function buildArtifacts(networkRecords) {
+  const frameUrl = networkRecords[0].url;
+  const trace = createTestTrace({
+    frameUrl,
+    timeOrigin: 0,
+    topLevelTasks: [{ts: 1000, duration: 50}],
+    largestContentfulPaint: 5000,
+    firstContentfulPaint: 2000,
+  });
+  const devtoolsLog = networkRecordsToDevtoolsLog(networkRecords);
+
+  return {
+    LinkElements: [],
+    URL: {
+      requestedUrl: baseResources[0].url,
+      mainDocumentUrl: baseResources[0].url,
+      finalDisplayedUrl: baseResources[0].url,
+    },
+    devtoolsLogs: {defaultPass: devtoolsLog},
+    traces: {defaultPass: trace},
+    GatherContext: {gatherMode: 'navigation'},
+  };
+}
 
 describe('Resources are fetched over http/2', () => {
   let artifacts = {};
   let context = {};
 
   beforeEach(() => {
     context = {settings: {throttlingMethod: 'simulate'}, computedCache: new Map()};
-
-    artifacts = {
-      traces: {defaultPass: trace},
-      devtoolsLogs: {defaultPass: devtoolsLog},
-      GatherContext: {gatherMode: 'navigation'},
-      URL: getURLArtifactFromDevtoolsLog(devtoolsLog),
-    };
+    artifacts = buildArtifacts(baseResources);
   });
 
   it('should pass when resources are requested via http/2', async () => {
```

Comment:
This test case isn't actually verifying that http/2 requests are ignored. It's only passing because there are <6 failing requests. As per my other comment, I think we should just ditch the `baseRecords` and create a new `networkRecords` list here that is 6 items long.

Reply:
ooops nice catch!